components/NotificationPopup/index.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from "react";
+
+import Image from "next/image";
+
+import classNames from "classnames/bind";
+
+import { useGetAllNotification } from "@/hooks/useGetAllNotification";
+
+import styles from "./NotificationPopup.module.scss";
+import BaseNotification from "../Notification/BaseNotification";
+import EmptyNotification from "../Notification/EmptyNotification";
+
+const cn = classNames.bind(styles);
+
+export default function NotificationPopup() {
+  const [isClicked, setIsClicked] = useState(false);
```

Comment:
```suggestion
  const [isPopupView, setIsPopupView] = useState(false);
```
P5: `isPopupView`와 같이 조금 더 명확하게 네이밍을 지어도 좋을 것 같아요~!